app/main.py

```diff
@@ -113,8 +116,55 @@ async def upload_image(file: UploadFile = File(...)):
         ])
 
 
-@app.get("/track_growth")
-async def track_growth():
-    raise HTTPException(
-        status_code=status.HTTP_501_NOT_IMPLEMENTED
-    )
+@app.get("/track_growth/")
+async def track_growth(center: Tuple[float, float] = None, scan_range: float = 360, day_range: int = 1000):
+
+    if center is None:
+        center = (0, 0)
```

Comment:
This is equivalent to making this a default in the function args

Reply:
Ah my bad I thought tuples were mutable but they're not, fixed :)